pkg/config/config_template.yaml

```diff
@@ -806,10 +806,25 @@ network_tracer_config:
 # Network tracer specific settings
 #
 # network_tracer_config:
-#   A boolean indicating the enabled state of the network tracer.
-#   enabled: true
-#   The full path to the file where network-tracer logs will be written.
+#   # A boolean indicating the enabled state of the network tracer.
+#   network_tracing_enabled: "true"
```

Comment:
Why is this renamed, and why is it quoted (instead of a real yaml bool)

Reply:
All flags are string to support "yes", "no".. and the renaming is a fix, in `stackstate-process-agent` this is the name of the flag.